src/components/EtaxForm/components/EqualLayoutForm.jsx

```diff
@@ -0,0 +1,53 @@
+import React from "react"
+import TextField from "material-ui/TextField"
+import PropTypes from "prop-types"
+
+/**
+ * EqualLayoutForm is a general purpose
+ * group of inputs that equally spread layout.
+ * @PureComponent
+ */
+const EqualLayoutForm = props => (
+  <div className="row col-12">
+    {props.fields.map(field => (
+      <div className="col-4" key={ field.id }>
+        <TextField
+          className="col-12"
+          id={ field.id }
```

Comment:
Also, if you wanted to pass every single field as a prop of the TextField element, you could use the rest and spread syntax, like so:

```
{fields.map(({id, ...field}) => (
  <div key={id}>
    <TextField
       id={id}
       onChange={props.handleChange(id)}
       {...field}
     />
  </div>
)}
```